test/e2e/federated-service.go

```diff
@@ -56,7 +56,7 @@ const (
 	FederatedDNS1123Regexp = `([a-z0-9]([-a-z0-9]*[a-z0-9])?\.)*([a-z0-9]([-a-z0-9]*[a-z0-9])?)`
 )
 
-var _ = framework.KubeDescribe("Service [Feature:Federation]", func() {
+var _ = framework.KubeDescribe("Federated Services", func() {
```

Comment:
The advantage is that the test name becomes:

"Federated Services should be discoverable by pods in the local cluster [Feature:Federation]"

rather than 

"Service [Feature:Federation] should be able to discover a federated service"

which doesn't describe what the test does, is not grammatically correct, and is difficult to read.

Yes, everything else works exactly as before this change.